### DISCUSSION TITLE
How to build a nsis app(Windows  installer)

### DISCUSSION THREAD

**Discussion by Xwudao** [cawa-93/vite-electron-builder#345] [Q&A] [answered]
I tried to edit `electron-builder.config.js` file, and add a `nsis` option, but when i run the compile command, it also only get an unpackaged dist.

How Can should i do?

**Comment by cawa-93** [comment, marked as answer]
In `compile` script used `--dir` perameter to build unpacked app and `--config.asar=false"` to disable asar arhiving. 
https://github.com/cawa-93/vite-electron-builder/blob/4e6b5561f9ff869bf7d485c379a900e1ad03b23f/package.json#L13

This is done intentionally to increase the speed of compiling on the local machine. I assume that compiling on a local machine is needed only for debugging.

**Reply by Xwudao** [reply]
What's my mean is that I want to  get an packaged app with windows installer rather than an unpack app.

such as:

![image](https://user-images.githubusercontent.com/31327868/126290691-20a04aba-59a4-4a2c-955c-ac02926b4f64.png)

**Reply by cawa-93** [reply]
Yeah. For that you need remove `--dir` parameter

**Reply by Xwudao** [reply]
thank you very much!!!
